docs/docs/apis/actions/external-authentication.md

```diff
@@ -2,7 +2,7 @@
 title: External Authentication Flow
 ---
 
-This flow is executed if the user logs in using an [identity provider](/guides/integrate/identity-providers) or using a [jwt token](/guides/integrate/identity-providers/jwt-idp).
+This flow is executed if the user logs in using an [identity provider](/guides/integrate/identity-providers/introduction) or using a [jwt token](/guides/integrate/identity-providers/jwt_idp).
```

Comment:
I think this is not right, it will not run when using a jwt token but when using a jwt idp, but thats just another type of idp

Reply:
Changed.